plugins/jobs/web_client/views/JobListWidget.js

```diff
@@ -190,9 +192,12 @@ var JobListWidget = View.extend({
     },
 
     _renderData: function () {
-        var jobs = this.collection.toArray();
+        if (!this.$('.g-main-content').length) {
+            // Do nothing until render has been called
+            return;
+        }
 
```

Comment:
Yes, but I think it's much safer not to rely on the that idiosyncrasy. Many other non-top-level Views don't self-render, and I wouldn't want this to be a source of bugs if that ever changes here.